server/models/UserRequests.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const userRequestsSchema = new mongoose.Schema({
+  user_id: {
+    type: Number,
+    required: true,
+    unique: true
+  },
+  sitter_id: {
+    type: Number,
+    required: true,
+    unique: true
+  },
```

Comment:
These could instead be references to the `User` model? Check out [this doc](https://mongoosejs.com/docs/populate.html) for more details.